src/encoded/types/__init__.py

```diff
@@ -278,6 +281,22 @@ class MicroscopeConfiguration(Item):
 
     item_type = 'microscope_configuration'
     schema = load_schema('encoded:schemas/microscope_configuration.json')
+    STATUS_ACL = {
+        'released'              : ALLOW_OWNER_EDIT + ALLOW_CURRENT,
+        'deleted'               : ALLOW_OWNER_EDIT + DELETED,
+        'draft'                 : ALLOW_OWNER_EDIT + ONLY_ADMIN_VIEW,
+        'released to lab'       : ALLOW_OWNER_EDIT + ALLOW_LAB_SUBMITTER_EDIT,
+        'released to project'   : ALLOW_OWNER_EDIT + ALLOW_VIEWING_GROUP_VIEW
+    }
+
+    def _update(self, properties, sheets=None):
```

Comment:
Generally once we add additional functions i.e. _update or calcprops other than a simple display_title we move the type information for an item and collection to it's own types file and write tests.

Reply:
@aschroed Sure, I'll move it to its own types file. (Once integration gets started in November, the pseudo-design was already implemented in init.py and continued as is.)

Reply:
I think deleted remaining editable by owner makes sense. If a user deletes a higlass display or micrometa, they can retrieve it back. If we need it really deleted, we can claim ownership as admin.

Reply:
Moved MicroscopeConfiguration to its own type file and add new unit tests. ready to be merged.